clients/fossid-webapp/src/main/kotlin/model/identification/markedAsIdentified/MarkedAsIdentifiedFile.kt

```diff
@@ -21,14 +21,15 @@ package org.ossreviewtoolkit.clients.fossid.model.identification.markedAsIdentif
 
```

Comment:
Commit message nit: "...problematic, as it introduce*s*..."
Occurs in multiple commit messages.